controllers/fenceagentsremediation_controller.go

```diff
@@ -222,9 +234,22 @@ func (r *FenceAgentsRemediationReconciler) Reconcile(ctx context.Context, req ct
 	if meta.IsStatusConditionTrue(far.Status.Conditions, v1alpha1.FenceAgentActionSucceededType) &&
 		!meta.IsStatusConditionTrue(far.Status.Conditions, commonConditions.SucceededType) {
 		// Fence agent action succeeded, and now we try to remove workloads (pods and their volume attachments)
-		r.Log.Info("Manual workload deletion", "Fence Agent", far.Spec.Agent, "Node Name", req.Name)
-		if err := utils.DeleteResources(ctx, r.Client, req.Name); err != nil {
-			r.Log.Error(err, "Manual workload deletion has failed", "CR's Name", req.Name)
+		switch far.Spec.RemediationStrategy {
+		case v1alpha1.ResourceDeletionRemediationStrategy:
+			r.Log.Info("Resource deletion - manually deleting workload", "Fence Agent", far.Spec.Agent, "Node Name", req.Name)
```

Comment:
See my other comment on the OutOfService log message below.

`Resource deletion - manually deleting workload` -> `Remediation strategy is resource deletion which explicitly deletes resources - manually deleting workload`.